src/main/java/org/isf/examination/rest/ExaminationController.java

```diff
@@ -268,4 +226,42 @@ public ResponseEntity<List<PatientExaminationDTO>> getByPatientId(@PathVariable
 			return ResponseEntity.ok(listPatientExaminationDTO);
 		}
 	}
+	
+    public void validateExamination(PatientExaminationDTO newPatientExamination) throws OHServiceException {
+	ExaminationParameters.initialize();
+		
+	if (newPatientExamination.getPex_height() < ExaminationParameters.HEIGHT_MIN || newPatientExamination.getPex_height() > ExaminationParameters.HEIGHT_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("The size should be between 0 and 250."));
+        }
+        if (newPatientExamination.getPex_weight() < ExaminationParameters.WEIGHT_MIN || newPatientExamination.getPex_weight() > ExaminationParameters.WEIGHT_MAX) {
+        	throw new OHAPIException(new OHExceptionMessage("The weight should be between 0 and 200."));
+        }
+        if (newPatientExamination.getPex_ap_min() < ExaminationParameters.AP_MIN_INIT) {
+        	throw new OHAPIException(new OHExceptionMessage("The minimum blood pressure must be at least 80."));
+        }
+        if (newPatientExamination.getPex_ap_min() > newPatientExamination.getPex_ap_max() ) {
```

Comment:
The error message should include the same MIN and MAX (or INIT) values.

Why we don't leverage the validation in core?

Reply:
I think it is because the error message return by this method is only use on the api if I add this methode on the core it will implies to update also the bundle file of gui and I don't have access to that file.

Reply:
yes you are right, we should totally avoid the use of bundles in the core... I will study how to do it.

as for now, please make `OHExceptionMessages` to include the parameters used in the comparations.